pkg/segment/structs/evaluationstructs.go

```diff
@@ -167,10 +167,17 @@ type NumericExpr struct {
 	Value        string
 
 	// Only used when IsTerminal is false.
-	Op    string // Including arithmetic, mathematical and text functions ops
-	Left  *NumericExpr
-	Right *NumericExpr
-	Val   *StringExpr
+	Op           string // Including arithmetic, mathematical and text functions ops
+	Left         *NumericExpr
+	Right        *NumericExpr
+	Val          *StringExpr
+	RelativeTime RelativeTimeExpr
+}
+
+type RelativeTimeExpr struct {
+	Snap     string
+	Offset   int64
+	TimeUnit utils.TimeUnit
 }
```

Comment:
I think this has exactly the same data as `RelativeTimeModifier`. Can we use that instead, or does it make sense to define this new type you have?

Reply:
Yes, `RelativeTimeExpr` contains the same data as `RelativeTimeModifier` from the `ast` package. However, the time modifiers are used as part of the initial search and may have more options when we support chained time modifiers in the future. The new `RelativeTimeExpr` type is specifically for the `relative_time` command and is not related to Time Modifiers.

I thought having separate types for each would make it easier to manage if we add more options to either type in the future. `RelativeTimeModifier` is used in the initial pipe search, while `RelativeTimeExpr` is used for evaluation.